tests/ui/test_create_pool_block_pool.py

```diff
@@ -111,7 +107,20 @@ def test_create_delete_pool(
         )
 
         # Getting IO results
-        get_fio_rw_iops(self.pod_obj)
+        self.get_fio_rw_iops(self.pod_obj)
+
+        # Checking the raw capcity is loaded on the UI or not.
+        blockpool_ui_object = BlockPoolUI(setup_ui)
+        if not blockpool_ui_object.pool_raw_capacity_loaded(self.pool_name):
+            raise BlockPoolRawCapacityNotLoaded(
```

Comment:
Why not raise NoSuuchElementException?

Reply:
Because ideally that element should be present in the DOM and its not present here, thus I went with the name BlockPoolCapacityNotLoaded exception.